plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -235,4 +243,32 @@ public IcebergConfig setRemoveOrphanFilesMinRetention(Duration removeOrphanFiles
         this.removeOrphanFilesMinRetention = removeOrphanFilesMinRetention;
         return this;
     }
+
+    public DataSize getTargetMaxFileSize()
+    {
+        return targetMaxFileSize;
+    }
+
+    @LegacyConfig("hive.target-max-file-size")
+    @Config("iceberg.target-max-file-size")
```

Comment:
If we're changing config properties, we should also document this change. @homar can you submit a follow-up PR with docs changes, or would you like me to take the lead on that?

cc @findepi

Reply:
I will submit a pr. Thanks for pointing it out

Reply:
@colebow  I made a PR https://github.com/trinodb/trino/pull/12619